doc/cug.tex

```diff
@@ -2822,9 +2822,13 @@ \subsubsection{Graph Types}
             graph = "delayed_cold_corge => corge"
 
         # Repeat once at the final cycle point
-        [[[ R1/P0Y ]]]
+        [[[ R1/$ ]]]
```

Comment:
I've added comments showing which recurrence form the headings conform to in the caret/dollar notation section, I can do this for the main list of examples as well if desired (though the recurrence forms aren't explained until after the list).